http-client/src/main/java/io/airlift/http/client/HttpClientModule.java

```diff
@@ -137,7 +142,7 @@ public HttpClient get()
                 ioPoolProvider = injector.getInstance(JettyIoPoolManager.class);
             }
 
-            JettyHttpClient client = new JettyHttpClient(config, ioPoolProvider.get(), ImmutableList.copyOf(filters));
+            JettyHttpClient client = new JettyHttpClient(config, kerberosConfig, Optional.of(ioPoolProvider.get()), ImmutableList.copyOf(filters));
```

Comment:
Yes, I made the constructor public and called it here (instead of creating yet another constructor for this case)